source/broadband/webpa_notification.c

```diff
@@ -468,6 +477,13 @@ static void getNotifyParamList(const char ***paramList, int *size)
         }
         count = count-1;
     }
+
+    // Remove Advanced Security params from NotifyList when cujo/fp is not enabled.
+    if(fpFlag == 0)
```

Comment:
fpFlag Default is 0 so worst case if syscfg fails parameters will be removed from notify. This is not good, do the reverse and check for 1 instead of default 0

Reply:
If syscfg fails also parameters should be removed from notify as it is already checked fp enable. 
i.e. Whenever fp is disabled then only it will go for syscfg case and in that case we need to remove from notify, so current changes looks fine.